### PR TITLE
PP-1158: TestQmgr fails saying "No such file or directory"

### PR DESCRIPTION
**Pull request by agrawalravi90**
#### Issue-ID

* **[PP-1158](https://pbspro.atlassian.net/browse/PP-1158)**

#### Problem description
* TestQmgr PTL test would fail saying "OSError: No such file or directory"

#### Cause / Analysis
* The test was assuming that qmgr was in the system's PATH variable, so when it was run on a system where that was not the case, it would fail not being able to find the qmgr executable.

#### Solution description
* Corrected the test case to now use the full path to qmgr

#### Checklist:

- [X] I have joined the **[pbspro community forum](http://community.pbspro.org/)**.
- [X] My pull request contains a **single, signed** commit. See **[setting up gpg signature](https://pbspro.atlassian.net/wiki/display/DG/Signing+Your+Git+Commits)**.
- [X] My code follows the **[coding style](https://pbspro.atlassian.net/wiki/display/DG/Coding+Standards)** of this project.
- [ ] My change requires project documentation. See **[required documentation checklist](https://pbspro.atlassian.net/wiki/display/DG/Checklist+for+Developing+Features+and+Bug+Fixes)** for details.
- [ ] I have added documentation in the **[project documentation area](https://pbspro.atlassian.net/wiki/display/PD)**.
- [ ] I have added new **PTL test(s) to my commit**. (See **[using PTL for testing](https://pbspro.atlassian.net/wiki/display/DG/Using+PTL+for+Testing)**) *(or)*
- [ ] I have added  **manual test(s) to the Jira ticket and explained why PTL is not appropriate** for this case.
- [X] All new and existing automated tests have passed. (See **[running automated PTL tests](https://pbspro.atlassian.net/wiki/display/DG/PTL+Quick+Start+Guide)**).
- [X] I have attached **test logs to the Jira ticket** as evidence of testing/verification.

__***For further information please visit the [Developer Guide Home](https://pbspro.atlassian.net/wiki/display/DG/Developer+Guide+Home).***__

### CODE REVIEW

File: test/tests/functional/pbs_qmgr.py [hunk 1]
```diff
@@ -76,13 +76,17 @@ def test_listcmd_whitespaces(self):
         node_prefix = "vn"
         nodename = node_prefix + "[0]"
         vndef_file = None
+        qmgr_path = os.path.join(self.server.pbs_conf["PBS_EXEC"], "bin",
+                                 "qmgr")
+        self.assertTrue(os.path.isfile(qmgr_path))
```
Comment (bhroam) [inline test/tests/functional/pbs_qmgr.py]: Don't assert here.  It will fail the test if qmgr isn't available.  Skip the test instead.

**Comment by agrawalravi90** [comment]
Thanks Bhroam, addressed your comment and uploaded the new logs to ticket. Please let me know if things look fine now.

@anamikau requesting you to check out the new changes as well. Thanks!

File: test/tests/functional/pbs_qmgr.py [hunk 2]
```diff
@@ -78,7 +78,8 @@ def test_listcmd_whitespaces(self):
         vndef_file = None
         qmgr_path = os.path.join(self.server.pbs_conf["PBS_EXEC"], "bin",
                                  "qmgr")
-        self.assertTrue(os.path.isfile(qmgr_path))
+        if not os.path.isfile(qmgr_path):
+            self.server.skipTest("Path to qmgr not found!")
```
Comment (bhroam) [inline test/tests/functional/pbs_qmgr.py]: It isn't the path to qmgr that isn't found, it's qmgr itself that isn't found

**Comment by agrawalravi90** [comment]
Thanks Bhroam. @anamikau , could you please look at the latest commit and sign off again?

**Comment by agrawalravi90** [comment]
Thanks a lot Anamika, I'll squash and rebase for the merge

**Pull request merged**